src/util/Locale.js

```diff
@@ -0,0 +1,153 @@
+import ObjectUtil from './Object.js'
+import UrlUtil from './Url.js';
+
+/**
+ * Locale related utility methods.
+ */
+const LocaleUtil = {
+
+  /**
+   * Hardcoded fallback if no supported languages are declared in app config.
+   */
+  supportedLanguageFallback: {'en': 'English'},
+
+  /**
+   * Import a webpack context for language files and returns message content.
+   * @param {Object} context The webpack context returned by require.context.
+   * @param {Object} mappingFunc A function to extract messages from an item of the webpack context.
+   *
+   * @returns {Object} A container with message data. Key is the language code, value contains the messages.
+   */
+  importLocales (context, mappingFunc) {
+    return context
+      .keys()
+      .map((key) => ({ key, locale: key.match(/[a-z0-9-_]+/i)[0] }))
+      .reduce(
+        (messages, { key, locale }) => ({
+          ...messages,
+          [locale]: mappingFunc(context(key))
+        }),
+        {}
+      );
+  },
+
+  /**
+   * Import all Wegue core language files from 'src/locales' and optionally
+   * app specific language files from 'app/locales'. Language files will be merged,
+   * such that Wegue core messages can be overridden by app messages.
+   *
+   * @returns A container with message data. Key is the language code, value contains the messages.
```

Comment:
I guess the output type is an `Obejct`

```suggestion
   * @returns  {Object} A container with message data. Key is the language code, value contains the messages.
```